<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serienviewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <item row="0" column="0" colspan="5">
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <widget class="QLabel" name="filterLabel">
        <property name="text">
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchEdit"/>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="deleteButton">
      <property name="toolTip">
       <string>Delete a serie from the list but of course keep it on the harddisk</string>
      </property>
      <property name="text">
       <string>remove from List</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/button_delete.png</normaloff>:/icons/button_delete.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="addButton">
      <property name="toolTip">
       <string>Add a serie to the list</string>
      </property>
      <property name="text">
       <string>add Serie</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="4" rowspan="2">
     <layout class="QVBoxLayout" name="sideBarLayout">
      <item>
       <widget class="QPushButton" name="playNextInSerieButton">
        <property name="text">
         <string>Play next in previous Serie (P)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="playLastAddedButton">
        <property name="text">
         <string>Play last added (L)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playNextButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Play next (N)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="undoButton">
        <property name="text">
         <string>Undo last Episode (U)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>9999999</width>
          <height>9999999</height>
         </size>
        </property>
        <property name="text">
         <string>If you click play the value for this episode immediately changed, please correct this, if you didn't watch the episode</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Next Serie:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="nextLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="nextNameLabel">
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Number of Series loaded:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="numberLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="addDirRecursiveButton">
      <property name="text">
       <string>add Dir Recursive</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="SerieView" name="tableView">
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::IgnoreAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSerieviewer">
    <property name="title">
     <string>Serieviewer</string>
    </property>
    <addaction name="actionAdd_recursive"/>
    <addaction name="actionSettings"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionClean_up_Series"/>
    <addaction name="actionQuit_without_saving"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSerie">
    <property name="title">
     <string>Serie</string>
    </property>
    <addaction name="actionShow_Series_Info"/>
    <addaction name="actionRewind_by_one"/>
    <addaction name="actionSet_the_link"/>
    <addaction name="actionSet_player"/>
   </widget>
   <widget class="QMenu" name="menuPlaying">
    <property name="title">
     <string>P&amp;laying</string>
    </property>
    <addaction name="actionRandom"/>
    <addaction name="actionNewRandom"/>
    <addaction name="actionPlay_next_in_Series"/>
    <addaction name="actionPlay_last_added"/>
    <addaction name="actionPlay_next"/>
    <addaction name="actionUndo_last_play"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSerieviewer"/>
   <addaction name="menuSerie"/>
   <addaction name="menuPlaying"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>a&amp;bout</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
  </action>
  <action name="actionClean_up_Series">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/cleanup.png</normaloff>:/icons/cleanup.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clean up Series</string>
   </property>
  </action>
  <action name="actionQuit_without_saving">
   <property name="text">
    <string>&amp;Quit without saving</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Q&amp;uit</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="actionShow_Series_Info">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Show Series Info</string>
   </property>
  </action>
  <action name="actionEnable_Ongoing">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/button_ok.png</normaloff>:/icons/button_ok.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Enable Ongoing</string>
   </property>
  </action>
  <action name="actionDisable_Ongoing">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/ongoing.png</normaloff>:/icons/ongoing.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disable Ongoing</string>
   </property>
  </action>
  <action name="actionRewind_by_one">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/previous.png</normaloff>:/icons/previous.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rewind by one</string>
   </property>
  </action>
  <action name="actionSet_the_link">
   <property name="text">
    <string>Set the &amp;link</string>
   </property>
  </action>
  <action name="actionSet_player">
   <property name="text">
    <string>&amp;Change Player/arguments</string>
   </property>
  </action>
  <action name="actionRandom">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/shuffle.png</normaloff>:/icons/shuffle.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pick Random</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionNewRandom">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/shuffle.png</normaloff>:/icons/shuffle.png</iconset>
   </property>
   <property name="text">
    <string>Pick &amp;New Random</string>
   </property>
  </action>
  <action name="actionPlay_next_in_Series">
   <property name="text">
    <string>Play next &amp;in Series</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionPlay_last_added">
   <property name="text">
    <string>Play &amp;last added</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionPlay_next">
   <property name="text">
    <string>Pla&amp;y next</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionUndo_last_play">
   <property name="text">
    <string>&amp;Undo last play</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>&amp;about Qt</string>
   </property>
  </action>
  <action name="actionAdd_recursive">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Recursive</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SerieView</class>
   <extends>QTableView</extends>
   <header>src/serieview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
